packages/backend-next/src/index.ts

```diff
@@ -44,5 +45,6 @@ backend.add(import('@backstage/plugin-search-backend-module-techdocs/alpha'));
 backend.add(import('@backstage/plugin-search-backend/alpha'));
 backend.add(import('@backstage/plugin-techdocs-backend/alpha'));
 backend.add(import('@backstage/plugin-todo-backend'));
+backend.add(jenkinsPlugin());
```

Comment:
Please use the `backend.add(import(...))` pattern like the lines above

Reply:
Updated as per comment